01_fib/01_fib.go

```diff
@@ -0,0 +1,30 @@
+
+package main
+
+import (
+"fmt"
```

Comment:
Indentation needs fixing. If you're using VSCode, the Go extension should autocorrect many such problems on save.

Reply:
Done